tests/bsim/bluetooth/mesh/tests_scripts/large_comp_data/get_comp2_data_split .sh

```diff
@@ -19,11 +19,25 @@ source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
 #    comp data with correspending bytes in local comp data.
 # 6. Client merges the two samples and checks that the collected data is
 #    correctly merged, continuous, and matches its local comp data.
+# Note: The test will run one iteration where the composition data page is in
+#       regular memory, and another for when it is stored in flash during FW
+#       update.
 conf=prj_mesh1d1_conf
-RunTest mesh_lcd_test_split_comp_data \
-	lcd_cli_split_comp_data_request lcd_srv_comp_data_status_respond
+overlay=overlay_pst_conf
```

Comment:
I'm uncertain if we need to change these tests to run with settings subsystem. Was there a particular need for this? We can always instantiate a separate `bst_test_instance` instance in the same test file wrapped into `if defined(CONFIG_BT_SETTINGS)` and added to `overlay_pst_conf`.

Reply:
Basically I need a functioning flash so that I can verify that the parsing functionality for fetching comp data pages from persistent storage works as expected. To do so I need the settings subsystem right?

Reply:
Ok, sorry, I missed that you changed old tests. Than it is fine!